zuul.d/jobs.yaml

```diff
@@ -2189,5 +2189,5 @@
       - Project:apache/hive
       - Application:Hive@master
       - OS:ubuntu-xenial
-      - Arch:aarch64
+      - Arch:arm64
```

Comment:
And here, looks duplicated with https://github.com/theopenlab/openlab-zuul-jobs/pull/588

Reply:
Yep, I will rebase if #588 is merge first.